opl/cluster_read.py

```diff
@@ -373,6 +373,42 @@ def measure(self, ri, name, command, output="text"):
         return name, result
 
 
+class CountLinePlugin(BasePlugin):
+    def measure(self, ri, name, command, output="text"):
+        """
+        Execute command "command" and return result as per its "output" configuration
+        """
+        error_count = 0
+        warning_count = 0
+        line_count = 0
+        
+        # Execute the command
+        result = execute(command)
+        result_lines = result.splitlines()
+
+        for line in result_lines:
+            line_count+=1
+            if line["level"] == "error":
```

Comment:
Do not assume log line is JSON, you need to use `re` module to match it with provided regexp.